src/Athens-Examples/AthensTextRenderManualTests.class.st

```diff
@@ -89,7 +90,7 @@ AthensTextRenderManualTests class >> test3 [
 			canvas pathTransform translateBy: 0 @11.
 		].
 	].
-	Display getCanvas translucentImage: surf asForm at: 100@100
+	surf asForm asMorph openInWindow
```

Comment:
Could it be

`surf asMorph openInWindow`?

Or do we want that in general surfaces can be transformed into forms?